huaweicloud/resource_huaweicloud_obs_bucket.go

```diff
@@ -85,6 +85,12 @@ func resourceObsBucket() *schema.Resource {
 				},
 			},
 
+			"quota": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  0,
```

Comment:
is that OK to have a 0 default value here?

Reply:
@niuzhenguo 
https://support.huaweicloud.com/intl/en-us/api-obs/obs_04_0052.html

Per documentation default, the value is 0. Should we omit that and make it implicit by OBS API rather than explicit default in terraform?

Reply:
@ShiChangkuo done.